src/AvtaleContext.tsx

```diff
@@ -230,7 +231,10 @@ export class TempAvtaleProvider extends React.Component<any, State> {
 
     async lagreAvtale() {
         const nyAvtale = await RestService.lagreAvtale(this.state.avtale);
-        this.setState({ avtale: nyAvtale, ulagredeEndringer: false });
+        this.setState({
+            avtale: { ...this.state.avtale, ...nyAvtale },
```

Comment:
Dette gjøres for å ikke overskrive avtale objektet med det som kommer fra backend. Dvs. `tiltakstype`-feltet i `avtale` objektet settes i frontend til `ARBEIDSTRENING` som default. kommer det et objekt fra backend som ikke har dette feltet så skjer det ingenting med dette feltet. Dette gjør at det vil være støtte for å tilstanden til backend både før og etter lønnstilskudd er implementert.